doc/en/installation/common/install_packages.rst

```diff
@@ -87,9 +87,9 @@ Run the commands::
     **centreon-database** package installs a database server optimized for use with Centreon.
 
 .. note::
-    Centreon does **not** support the SQL STRICT mode yet. Please make sure that
-    it is disabled. For more information on how to disable the mode please check
-    the official `MariaDB documentation <https://mariadb.com/kb/en/library/sql-mode/#strict-mode>`_.
+    Centreon **does** support the SQL STRICT mode. For more information about it please check
+    the official `MariaDB documentation <https://mariadb.com/kb/en/library/sql-mode/#strict-mode>`_. Or the official
+     `MySQL documentation <https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html>`_.
```

Comment:
I think we can already remove this information, right? :thinking:

Reply:
Maybe, it would be better, if we let the link to the official documentation for users wanting to enable the StrictMode